src/search/rank/score.go

```diff
@@ -0,0 +1,23 @@
+package rank
+
+import (
+	"math"
+
+	"github.com/hearchco/hearchco/src/config"
+	"github.com/hearchco/hearchco/src/search/result"
+)
+
+func GetScore(result result.Result, rconf config.Ranking) float64 {
```

Comment:
This is hot codepath (performance important), its probably excusable to pass these as pointers here.